lib/foreman_maintain/concerns/metadata.rb

```diff
@@ -104,35 +138,15 @@ def metadata
       end
 
       def label
-        metadata[:label] || generate_label
+        self.class.label
```

Comment:
Afaik `class` is one of the things you can't use with implicit receiver, as it's also a keyword, so the interpreter considers it beginning of a class definition

```
class.to_s
SyntaxError: (irb):3: syntax error, unexpected '.'
```